### DISCUSSION TITLE
Kazuharu Yanagimoto - Some Quarto Tips for Academic Presentation

### DISCUSSION THREAD

**Discussion by giscus** [nicetak/nicetak.github.io#11] [General]
# Kazuharu Yanagimoto - Some Quarto Tips for Academic Presentation

https://kazuyanagimoto.com/blog/2022/12/27/quarto_tips/

**Comment by danm0nster** [comment]
Yes, I'm a Beamer fan, and now I am also a Quarto fan!

Thanks for this very nice and very useful blog post! I've been using Quarto to prepare slides for a new course in the spring of 2023, and I'm very happy, although there are some rough edges.

Your tables look really nice, but when I use `gt()` my tables are much smaller, and I constantly need to change the font size or the width to get nice output. I downloaded you example `slides.qmd`and `custom.scss` to see if I can reproduce your nice results, but even with your files I get a very small and unreadable table. Do you have any idea why?

BTW: There is a (slightly) easier way to get superscripts and subscripts in `gt()` tables, viz., by using the `md()` function. Here is an example:
```
model_comparison |> 
  gt() |> 
  cols_label(name = "Model", R2 = md("R^2^"),
             R2_adj = md("Adj. R^2^"), F = "F")
```

This might not work with your example inside `mutate()`, but is easier to type than the HTML tags.

**Reply by kazuyanagimoto** [reply]
Hi Dan, I'm glad you liked it and thanks for the comment!

Yes, I realized the issue of the font size yesterday. Previously, `gt`'s font size works fine in quarto slides without any settings,  but something seems changed (not sure whether it comes from quarto or gt updates.) 

My current solution is to define a theme of `gt` as `ggplot2`. 
```r
gt_theme_quarto <- function(data, ...) {
  data |>
    tab_options(
      table.font.size = "0.8em",
      ...
    )
}

# For every gt object
gtb |>
  gt_theme_quarto()
```

I'm now trying to find a nice custom theme, which matches to my slides themes. I will post another blog for that once I figure it out.

**Reply by danm0nster** [reply]
Thanks for the quick reply! A theme function is a clever solution.

I wonder if this is a bug that should be reported as an issue to the quarto or gt developers?

**Comment by dana89co** [comment]
Thanks for these Quarto tips! I have created a couple of presentations already and I think it does help a lot for academic presentations. However, I ran into the same issue with the hyperlink. Would love to know if there is a solution available.
